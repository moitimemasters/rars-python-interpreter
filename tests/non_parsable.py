if:
    a += 4
else:
    a += 5
