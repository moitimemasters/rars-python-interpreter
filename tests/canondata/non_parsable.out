file: tests/non_parsable.py
started lexing
ended lexing
started parsing ast
address of if condition: 56
ended parsing ast
started compiling
